pkg/logs/input/docker/tailer.go

```diff
@@ -135,10 +145,16 @@ func (t *Tailer) readForever() {
 			inBuf := make([]byte, 4096)
 			n, err := t.reader.Read(inBuf)
 			if err != nil {
+				if isClosedConnError(err) {
+					log.Warn("Closed conn")
+					return
+				}
 				// an error occurred, stop from reading new logs
 				if err != io.EOF {
 					t.source.Status.Error(err)
 					log.Error("Err: ", err)
+					log.Warn("The tailer of container ", ShortContainerID(t.ContainerID), " will restart")
```

Comment:
We could probably merge both messages to simplify a bit, eg. "Tailing container X stopped because of error Y"
Also logging the fact that we are (re)starting tailing should probably be done where we consume erroredContainerID.